examples/transpiler/babel.bzl

```diff
@@ -35,7 +43,16 @@ def babel(name, srcs, js_outs, map_outs, **kwargs):
                 "//examples:node_modules/@babel/preset-typescript",
             ],
             chdir = "examples",
-            outs = outs,
+            outs = [js_out, map_out],
             args = args,
             **kwargs
         )
+
+        outs.append(js_out)
+        outs.append(map_out)
+
+    # The target whose default outputs are the js files which ts_project() will reference
+    native.filegroup(
```

Comment:
note, I changed this from `js_library` to `filegroup` to be more clear you don't actually need any providers returned

Reply:
sounds good 👍